README.md

```diff
@@ -3,16 +3,9 @@
 [![GitHub Build Status](https://github.com/cisagov/ansible-role-crowdstrike/workflows/build/badge.svg)](https://github.com/cisagov/ansible-role-crowdstrike/actions)
 [![CodeQL](https://github.com/cisagov/ansible-role-crowdstrike/workflows/CodeQL/badge.svg)](https://github.com/cisagov/ansible-role-crowdstrike/actions/workflows/codeql-analysis.yml)
 
-This is a skeleton project that can be used to quickly get a new
-[cisagov](https://github.com/cisagov) Ansible role GitHub project
-started, where that Ansible role requires an AWS test user for
-automated testing.
-
-This skeleton project contains [licensing information](LICENSE), as
-well as [pre-commit hooks](https://pre-commit.com) and [GitHub
-Actions](https://github.com/features/actions) configurations
-appropriate for an Ansible role, as well as the Terraform code to
-create the AWS test user.
+This is an Ansible role for installing [CrowdStrike
```

Comment:
This aligns with the language CrowdStrike themselves use in the [installation guide](https://www.crowdstrike.com/blog/tech-center/install-falcon-sensor-for-linux/).

```suggestion
This is an Ansible role for installing the [CrowdStrike
```